### PR TITLE
Skipping ExtractTextPipeline if the item has no files

### PR DESCRIPTION
**Pull request by phrfpeixoto**
This is a simple fix for #256

### CODE REVIEW

File: processing/data_collection/gazette/pipelines.py [hunk 1]
```diff
@@ -34,6 +34,9 @@ def process_item(self, item, spider):
         if not extract_text_from_file:
             return item
 
+        if len(item["files"]) == 0:
```
Comment (ejulio) [inline processing/data_collection/gazette/pipelines.py:37]: I guess `not item["files"]` reads better, unless you want to check specifically for an empty list

**Review by ejulio — APPROVED** [review]

**Comment by jvanz** [comment]
Thanks for your PR @phrfpeixoto ! But I`ll not accept it. We are in the process of moving off the data manipulation code from this repository. So, this code will be obsolete within days. Sorry... =/

If you interesting in check the future commits on that. Keep eyes on this repo: https://github.com/okfn-brasil/querido-diario-data-processing

**Comment by phrfpeixoto** [comment]
Added another check for `SQLDatabasePipeline`, which also has a bug. And this time, no way to work around it.
@giuliocc @rennerocha @ejulio 
Can we re-open this PR?

File: processing/data_collection/gazette/pipelines.py (hunk 1 above)
Comment (phrfpeixoto) [reply processing/data_collection/gazette/pipelines.py:37]: Fixed.

**Comment by ejulio** [comment]
@phrfpeixoto , no new commits here, are you sure you pushed it?
Also, if the change is in another pipeline, why not making it a new PR?

**Comment by phrfpeixoto** [comment]
Will do

**Comment by giuliocc** [comment]
@phrfpeixoto for this pipeline, I think we should wait for #310. If there's a bug in the SQL pipeline I suggest opening a new PR for solving it.

**Pull request closed without merging**